api/src/pcapi/static/backofficev3/js/addons/pc-role.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Add filtering on description to search role input
+ *
+ * Using `data-search-target=".some-container"`, you can define the target container
+ * which have the roles input to filter.
+ *
+ * It automatically tries to expand all bootstrap collapse within target container on search.
+ *
+ * @example
+ * <form class="input-group my-3 px-1">
+ *    <span class="input-group-text bg-white"><i class="bi bi-search"></i></span>
+ *    <input type="text"
+ *           class="form-control col-8 border-start-0"
+ *           name="q"
+ *           value=""
+ *           data-search-role
+ *           data-search-target=".some-container"
+ *           aria-describedby="search-icon"
+ *           placeholder="Gérer les offres, ..."/>
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏idem au niveau du placeholder

Reply:
Ajout du suffix `Ex :`